eksctl/01_setup.md

```diff
@@ -0,0 +1,103 @@
+# eksctl setup
+
+<!-- TOC -->
+
+- [eksctl setup](#eksctl-setup)
+  - [Overview](#overview)
+  - [Tips](#tips)
+  - [Install eksctl](#install-eksctl)
+  - [Check your default region of AWS](#check-your-default-region-of-aws)
+  - [Requirements for instance](#requirements-for-instance)
+  - [Create cluster with node type](#create-cluster-with-node-type)
+    - [Create Cluster (ARM)](#create-cluster-arm)
+    - [Create Cluster  (x86)](#create-cluster--x86)
+  - [Get Cluster](#get-cluster)
+- [Delete Cluster](#delete-cluster)
+
+<!-- /TOC -->
+
+## Overview
+
+eksctl is a simple CLI tool for creating clusters on EKS - Amazon's new managed Kubernetes service for EC2. It is written in Go, and uses CloudFormation.
+
+eksctl is explained in a straightforward manner in [this video](https://youtu.be/p6xDCz00TxU).
+
```

Comment:
```suggestion
Undertaking these steps is estimated to cost around $70-$90 per month. However, this is an estimate and not a guaranteed figure. It is your responsible to manage the AWS cost. Instead of deleting every resource on AWS Console by your own, you can simply delete the cluster with eksctl command, and it will handle for you. You must double check if there any left overs that can cost you.
```